src/sparseml/pytorch/utils/exporter.py

```diff
@@ -208,17 +208,34 @@ def export_onnx(
         """
         if not export_kwargs:
             export_kwargs = {}
+
+        module = deepcopy(self._module).cpu()  # don't modify the original model
```

Comment:
I see we deepcopy in `TorchToONNX`, wondering if this will be too much overhead as models get larger, probably not, just something to keep in mind

Reply:
Yeah this could definitely add computation time for larger models, but I think its better to avoid modifying the original model thats passed in to the function. If a user is going to keep using the model after export we don't want to be unexpectedly changing it

Reply:
yes agreed - if we ever go back, we can always track which modules are updated and what their original ranges were